lib/veeqo/base.rb

```diff
@@ -0,0 +1,11 @@
+module Veeqo
+  class Base
+    def self.method_missing(method_name, *arguments, &block)
```

Comment:
We are going to ignore this for now, as if the class method does not exists as instance method then we are falling back to the `super`. So I guess we are okay here.